test/Database/LSMTree/Model/IO/Monoidal.hs

```diff
@@ -0,0 +1,90 @@
+{-# LANGUAGE TypeFamilies #-}
+
+-- | A @TVar@-based monoidal model of a potentially closed mutable session,
+-- giving an instance of `Class.IsTableHandle`.
```

Comment:
I don't think it's too important that it is using a `TVar`. I would emphasise that this is the pure `Session` model lifted into `IO`